### ISSUE TITLE
[Arcane Fractured Jinx(All Forms)] Passive Screen effects and Neon Purple Model Change Not working

### ISSUE THREAD

**Issue by ptspapaztr** [darkseal-org/lol-skins#105]
## Description
I tried to kill a champion in Practice Tool but the effects weren't complete.

## Affected Skin
- Skin Name: Arcane Fractured Jinx(All)

## Steps to reproduce 
1. Kill a champion
2. Look at to the characther model and compare with the actual one

## Expected Behavior
When a champion is killed there should be an effect on the champion model which turns her to a neon purple color and a screen effect counting the killing spree until pentakill.
## Actual Behavior
Passive screen effect and characther model turning into purple effect aren't present when a champion is killed.

## Screenshots (optional, but very important)

https://github.com/user-attachments/assets/f77dbe87-cc2a-49f4-a586-aacbcf9519d1

## Additional Details
- Custom Skin Application: CsLoL Manager
- Custom Skin Application version: 1.0
- League of Legends Version: 14.24
- Operating System: Windows 11

**Comment by siixx616** [comment]
hi, u can get banned for this? if i dont say anything they cant know right?

**Comment by siixx616** [comment]
There are 2 more missing things, the more important one is that the weapon should be different color in each form but is all pink, and the second one is that when u kill and enemy there is a music like pyke

**Comment by siixx616** [comment]
No, she not i dont have a problem but i could be cool to have it, i know it takes time and i really aprecciate the creators work. (heres a link to the video of the music after she kills 1, 2, 3, 4, 5 enemy champs)

**Comment by siixx616** [comment]
Really? i cant  hear it, i dont know what to do, anyways another thing i noticed is that the ult and the W sfx are the same as form 1 (with the hood one), i really wanted to say too that i really aprecciate the creators work and feedback, really cool, and i really do like this little things in skins and is really nice having it.

**Comment by siixx616** [comment]
I just noticed that they are fixing Jinx and evelynn skins, im so excited

**Comment by siixx616** [comment]
this is the music im talking about, maybe is really hard to implement or something https://www.youtube.com/watch?v=XlYwetGT_0A&t=3s

**Comment by tuananhcn** [comment]
> Really? i cant hear it, i dont know what to do. In the other side another thing i noticed and im sending to help is that ALL the sfx (sound effects) including Q rocket launcher autoattacks are the same from the (Hood form). I really wanted to say too that i really aprecciate the creators work and feedback, really cool, and i really do like this little things in skins and is really nice having it. edit: I just noticed that they are fixing Jinx and evelynn skins, im so excited

All her skill sound effect currently is hood form am i right ?

**Comment by siixx616** [comment]
Here are things i noticed (to help in the feedback and contribution)

-The Jinx interactions (when she talks) the should be different in each form, Powder is less agressive for example, but in the 3 forms are all the same interactions and from the hood form one
- Missing HUD effect when the passive is on (on kill)
- In the same passive/HUD effect there is a music that plays for every 1/2/3/4 and 5 kills, each one is different just like when Pyke is executed (i hope i explained well)
- The Sound Effects in Powder Form and green Form are the same sound effects as the Hood one (Q autoattacks, W, E and most noticeable one the ULT)
-And last one is the ctrl 5 not changing forms  but i dont think is possible since this is a fantome.

The recent changes are tattos, gun color are different in each form and the passive turns neon in the passive, thanks about that <3
